### ISSUE TITLE
Using IPFS in Native communities

### ISSUE THREAD

**Issue by arecvlohe** [ipfs/local-offline-collab#17]
## Summary

My name is Adam Recvlohe and I lead the @nativesintech GitHub organization and non-profit. Check out our website here: [website](https://nativesintech.org/). I have heard of IPFS for a while but today I stumbled upon it again and found it was pretty easy to get setup on my computer and in the browser. I will soon start hosting our website on IPFS following the instructions here: https://gist.github.com/claus/1287f47b5fbaaea338ac8a04d02bf258.

## Question

Intermittent or poor connection is a reality for some Native nations here in the US.  I would like to do a collaboration with IPFS that creates some training materials (blog posts, videos, cheat sheets, etc) that are directed at Native peoples with poor access to internet. Is this something this organization would be interested in? Additionally, I don't have a lot of experience with Offline First application models so if I could consult with you all on some of the best practices that would be appreciated as well. 

## Next Steps

If you are interested in working collaboratively with me on this project feel free to send me an email at hello@nativesintech.org.

**Comment by terichadbourne** [comment]
@arecvlohe Any chance you'd like to spend some time telling us about affect of poor connections on Native nations on our next monthly call on Wednesday, September 18? ( [Call details here](https://github.com/ipfs/local-offline-collab/issues/21).)

**Comment by arecvlohe** [comment]
@terichadbourne Sure! What time would that be?

**Comment by terichadbourne** [comment]
Fabulous, @arecvlohe. It's at 10am PDT | 1pm EDT | 5pm UTC on Wed 9/18. Will that work for you? 

If you have an email address connected to your calendar, I'd be happy to send an invite to get it on there. It's also listed on the [IPFS Community Calendar](https://calendar.google.com/calendar/embed?src=ipfs.io_eal36ugu5e75s207gfjcu0ae84@group.calendar.google.com&pli=1)

**Comment by arecvlohe** [comment]
Yes, please send an invite to adam.recvlohe@gmail.com. Thank you!
